### PR TITLE
Complete iteration 4

### PR DESCRIPTION
**Pull request by srslie**
- **What changes are you making?**
     - Added doubleclick to remove saved poster from array
     - Organized variables and functions
     - Got rid of newPoster and userPoster to use currentUser for all
     - Made helper functions toggle to show pages and displayPoster to populate display with poster object data

- **Do you have any questions or issues about this addition?**
     - How can we make this more efficient re: loops/refactoring?
     - How might we look into data validation or drag & drop functionality?

- [x] **Did you test your code locally first?**

- [x] **Does this follow our intended guidelines (DRY & commented) for project code?**

### CODE REVIEW

**Comment by gbarka2** [comment]
1000000%!!!!

File: src/main.js [hunk 1]
```diff
@@ -135,52 +122,53 @@ showMyPosterButton.addEventListener('click', showMyPoster);
...
-
 function formToggle() {
-    toggleHidden(posterForm);
-    toggleHidden(mainPoster);
+  toggleHidden(posterForm);
+  toggleHidden(mainPoster);
 }
-
 function savedToggle() {
   toggleHidden(mainPoster);
```
Comment (sschipke) [inline src/main.js:134]: Great job realizing you could create a reusable function for this!

File: src/main.js [hunk 2]
```diff
@@ -189,25 +177,17 @@ function savePoster() {
        <h4>${savedPosters[i].quote}</h4>
        </article>`
```
Comment (sschipke) [inline src/main.js:178]: Great use of interpolation!

File: src/main.js [hunk 3]
```diff
@@ -189,25 +177,17 @@ function savePoster() {
...
-  quote.innerText = userPoster.quote;
-  mainPoster.classList.remove("hidden");
-  posterForm.classList.add("hidden");
+}
+function displaySaved() {
+  savedToggle();
+  displayMiniPosters();
+}
+function deleteSavedPoster() {
+  var selectedID = event.target.closest(".mini-poster").id;
```
Comment (sschipke) [inline src/main.js:186]: 🚀

File: src/main.js [hunk 4]
```diff
@@ -135,52 +122,53 @@ showMyPosterButton.addEventListener('click', showMyPoster);
...
-
+function getRandomIndex(array) {
+  return Math.floor(Math.random() * array.length);
+}
+function buildNewDisplay(poster) {
+  image.src = poster.imageURL;
+  title.innerText = poster.title;
+  quote.innerText = poster.quote;
+}
+function displayPoster() {
```
Comment (sschipke) [inline src/main.js:145]: I would consider renaming this function. Is it displaying a _selected_ poster, or generating a random one?

**Pull request merged**